src/default_panels/GraphCreatePanel.js

```diff
@@ -59,11 +63,26 @@ const GraphCreatePanel = (props, {localize: _}) => {
         options={[{label: _('Vertical'), value: 'v'}, {label: _('Horizontal'), value: 'h'}]}
       />
       <HistogramInfoVertical>
-        {_('Note: in vertical orientation, X values are used for bins and Y values for weights.')}
+        {_(
+          'Note: in vertical orientation, X values are used for bins and Y values for weights. You can configure the binning function in the '
```

Comment:
wordsmithing ... :) "... X values are used for binning. If Y values are provided, they are used as inputs to the histogram function which you can configure in the Traces panel. If Y values are omitted, the histogram function defaults to Count."

Also please rename "binning function" to "histogram function" in the Traces panel.

Reply:
Also: please move the "histogram function" outside of the "binning" section, right below normalization.

Reply:
Done